### PR TITLE
Work-section

### PR DESCRIPTION
**Pull request by sj1978**
in this section I:
.Created a section with an H2.
.Created an image.
.Created a work section
Created a button.

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @sj1978, 

**This is a great project you have going on, well-done lad**. 
   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -71,6 +72,110 @@ <h2>Software Developer</h2>
...
+        </div>
+
+    </section>
+    <section class="work-sect">
+        <div>
+            <h2>MY RECENT WORK</h2>
+        </div>
+        <div class="work-sect">
+            <img class="card-image" src="/assets/images/dummy-image-24toolbox-350x450.png" alt="dummy image">
+            <h3>Multi-Post Stories</h3>
```
Comment (mke2111) [inline index.html:107]: - [ ] My Recent Work is the section title. You have repeated it for every project. You should only have one for the entire section

File: index.html [hunk 2]
```diff
@@ -71,6 +72,110 @@ <h2>Software Developer</h2>
         </div>
 
     </footer>
+    <section class="work-sect">
+        <div>
+            <h2>MY RECENT WORK</h2>
+        </div>
+        <div class="work-sect">
+            <img class="card-image" src="/assets/images/dummy-image-24toolbox-350x450.png" alt="dummy image">
+            <h3>Multi-Post Stories</h3>
```
Comment (mke2111) [inline index.html:81]: - [ ] All the images for the projects are not visible. Give this a fix.

File: index.html [hunk 3]
```diff
@@ -71,6 +72,110 @@ <h2>Software Developer</h2>
...
+                <li class="border2">Bootstrap</li>
+                <li>Ruby</li>
+            </ul>
+            <button class="btn">See Project</button>
+
+
+
+        </div>
+
+    </section>
```
Comment (mke2111) [inline index.html:152]: - [ ] For each project, ensure the section uses the font and colors background and font) defined for the template. Also put the margins (top, bottom, left, right) defined in the design into consideration.
Your Project
Screenshot from 2021-09-17 02-21-13

Original design
Screenshot from 2021-09-17 02-23-58

File: index.html [hunk 4]
```diff
@@ -58,7 +59,7 @@ <h2>Software Developer</h2>
             <li><a href="#">Terminal</a></li>
         </ul>
     </section>
```
Comment (mke2111) [inline index.html:61]: - [ ] For this entire section, make it match the given design.
Your page
Screenshot from 2021-09-17 02-27-25

Original page

Screenshot from 2021-09-17 02-29-16

**Review by rukundoeric — APPROVED** [review]
Hi @sj1978,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**